SlackerReferenceApp/bower.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "HelloIonic",
+  "private": "true",
+  "devDependencies": {
+    "ionic": "driftyco/ionic-bower#1.1.1"
```

Comment:
I'd keep the dependency, just add www/libs to .gitignore. That way users can install using npm install && bower install.